package/CHANGELOG

```diff
@@ -115,6 +115,8 @@ Enhancements
   * PersistenceLength.plot() now create new axes if current axes not provided (Issue #2590)
 
 Changes
+  * Use `np.clip()` to counter the rounding off error in `mdamath.angles` and 
```

Comment:
Just noticed this, 
a) The main change here is that you now enforce a return of `np.pi` rather than `-np.pi` when x is less than -1.0. That's probably what you want to detail here, i.e. "`mdamath.angles` now returns `np.pi` instead of `-np.pi` in cases of lower bound roundoff errors."

b) this probably should be under "Fixes" rather than "Changes".

c) as far as I remember we don't usually detail new tests in the changelog, soI'd probably just end the entry at `mdamath.angles` (followed by the Issue/PR number) as your entry is sufficiently decriptive 😃